cgmanifests/generated/cgmanifest.json

```diff
@@ -278,6 +278,16 @@
         "comments": "mp11"
       }
     },
+    {
```

Comment:
already an entry in line 102 of this file

Reply:
Because I didn't update the submodule.